File: src/main/java/org/srd/ediary/domain/model/Mood.java
package org.srd.ediary.domain.model;

import java.time.LocalDate;
import java.time.LocalDateTime;

public class Mood {
    private Long id;
    private Long ownerID;
    private int scoreMood;
    private int scoreProductivity;
    private LocalDateTime bedtime;
    private LocalDateTime wakeUpTime;
    private LocalDate createdAt;
}

File: frontend/ediary_cli/src/main/kotlin/org/srd/ediary_cli/model/Mood.kt
package org.srd.ediary_cli.model

import kotlinx.datetime.LocalDate
import kotlinx.datetime.LocalDateTime
import kotlinx.serialization.Serializable

@Serializable
data class MoodCreateDTO(
    val ownerID: Long,
    val scoreMood: Int,
    val scoreProductivity: Int,
    val bedtime: LocalDateTime,
    val wakeUpTime: LocalDateTime
)

@Serializable
data class MoodUpdateDTO(
    val scoreMood: Int,
    val scoreProductivity: Int,
    val bedtime: LocalDateTime,
    val wakeUpTime: LocalDateTime
)

@Serializable
data class MoodInfoDTO(
    val id: Long,
    val scoreMood: Int,
    val scoreProductivity: Int,
    val bedtime: LocalDateTime,
    val wakeUpTime: LocalDateTime,
    val createdAt: LocalDate
)
